Tests/Integration/CloudToDeviceMessageTests.cs

```diff
@@ -601,23 +572,22 @@ public async Task When_Device_Prefers_Second_Window_Should_Send_Downstream_In_Se
         [Theory]
         // Preferred Window: 1
         // - Aiming for RX1
-        [InlineData(1, 0, 400, 610, 1)] // 1000 - (400 - noise)
-        [InlineData(1, 100, 300, 510, 1)]
-        [InlineData(1, 200, 200, 410, 1)]
+        [InlineData(1, 0, 400, 610)] // 1000 - (400 - noise)
+        [InlineData(1, 100, 300, 510)]
+        [InlineData(1, 200, 200, 410)]
         // - Aiming for RX2
-        [InlineData(1, 750, 690, 999, 2)]
-        [InlineData(1, 1000, 250, 610, 2)]
+        [InlineData(1, 750, 690, 999)]
+        [InlineData(1, 1000, 250, 610)]
 
         // Preferred Window: 2
         // - Aiming for RX2
-        [InlineData(2, 0, 1400, 1610, 2)]
-        [InlineData(2, 100, 1300, 1510, 2)]
+        [InlineData(2, 0, 1400, 1610)]
+        [InlineData(2, 100, 1300, 1510)]
         public async Task When_Device_Checks_For_C2D_Message_Uses_Available_Time(
             int preferredWindow,
             int sendEventDurationInMs,
             int checkMinDuration,
-            int checkMaxDuration,
-            int expectedRX)
```

Comment:
this was removed as we can't decide when to send a message, this decision is made by the basicstation

Reply:
To be verified

Reply:
Add assert lnsrxdelay = 1